docs/_includes/Types.md

```diff
@@ -3472,6 +3490,8 @@ If the compiler does specialize the run-time representation for
 `int32`, then these statements come at the expense of two,
 respectively three, run-time conversions.
 
+TO BE WRITTEN - change of int(x) to x as int
+
 # Subset types
 ````
 NatType_ = "nat"
```

Comment:
There are more built-in subset types:

`nat` is a built-in subset type of `int`
`-->` is a built-in subset type of `~>`
`->` is a built-in subset type of `-->`
Each non-null type `C` is a built-in subset type of `C?`

(I think the line `NatType_ = "nat"` was added to the reference manual at a time when it was the only subset type. It was built-in and handled specially. 3 years ago, Dafny got user-defined subset types, so `nat` is now a built-in one of those.)